crates/cli/src/commands/order/orderbook.rs

```diff
@@ -0,0 +1,58 @@
+use crate::execute::Execute;
+use crate::output::{output, SupportedOutputEncoding};
+use anyhow::{anyhow, Result};
+use clap::Args;
+use rain_orderbook_app_settings::Config;
+use rain_orderbook_common::dotrain_order::DotrainOrder;
+use std::fs::read_to_string;
+use std::path::PathBuf;
+
+#[derive(Args, Clone)]
+pub struct Orderbook {
```

Comment:
this name is generic and confusing - should be OrderbookAddress

Reply:
Resolved